release/artifacts.yaml

```diff
@@ -1,25 +1,36 @@
-- target: //daml-lf/archive:daml_lf_archive_java
-  type: jar
-  javadoc-jar: daml_lf_archive_java_javadoc.jar
-  src-jar: daml_lf_archive_java_srcjar.jar
-  mavenUpload: True
 - target: //daml-lf/data:data
   type: jar-scala
-- target: //compiler/damlc/jar:damlc_jar
-  type: jar-deploy
-  platformDependent: True
-- target: //daml-lf/archive:daml_lf_archive_scala
-  type: jar-scala
-- target: //daml-lf/archive:daml_lf_archive_protos_zip
+- target: //daml-lf/archive:daml_lf_archive_java_proto
+  type: jar-proto
+  mavenUpload: True
+- target: //daml-lf/archive:daml_lf_archive_proto_zip
+  type: zip
+  location:
+    groupId: com.digitalasset
+    artifactId: daml-lf-archive-protos
+- target: //daml-lf/archive:daml_lf_archive_proto_tarball
+  type: targz
+  location:
+    groupId: com.digitalasset
+    artifactId: daml-lf-archive-protos
+- target: //daml-lf/archive:daml_lf_1.6_archive_java_proto
+  type: jar-proto
```

Comment:
What is the interaction between `jar-proto` and `mavenUpload`? Do we need to upload separate "javadoc" and "sources"?

Reply:
rvert back to something closer to the original